src/guide/reactivity-computed-watchers.md

```diff
@@ -215,7 +215,7 @@ lastName.value = 'Smith' // logs: ["John", "Smith"] ["John", ""]
 
 However, if you are changing both watched sources simultaneously in the same method, the watcher will be executed only once:
```

Comment:
This sentence is not valid haha

Reply:
Still valid for me, when joining changes in the same method, watcher is triggered only once 🤷🏻‍♀️